cdap-ui/app/cdap/components/FieldLevelLineage/LineageSummary/LineageSummary.scss

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import '../../../styles/variables.scss';
+$box-border-color: $orange-01;
+$box-bg-color: $cdap-white;
+$secondary-color: $grey-04;
+$section-border-color: $grey-03;
+$field-row-border-color: $grey-05;
+$field-row-hover-bg: $grey-08;
+
+.field-level-lineage-container {
+  .lineage-summary-container {
+    width: 250px;
+    border: 2px solid $box-border-color;
+    background-color: $box-bg-color;
+    position: absolute;
+    transform: translateX(-100%);
+
+    /*
+     * Left offset calculation
+     * 50% (middle of screen) - half of the field list box - margin
+     */
+    left: calc(50% - (250px / 2) - 15px);
```

Comment:
Same here. If we use flexbox and align and justify contents center we seem to be getting the same layout.

Reply:
not really, flexbox will align everything either at the start of the container, center, or middle. In this case we want the fields list to always be centered, and the lineage popover will be to the left of it. I don't think that can be achieved with flexbox.